### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sai-Nissankula**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/CHANGELOG.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
+  - RelativityOne current and preview versions
+  - Relativity Server 2022 (12.1) and later
+
+## Documentation
+
+For more details and common usage patterns check out [our documentation](https://relativitydev.github.io/relativity.testing.framework.api/).
+
+## Where to get help
+
+- For general help and questions, please start a [Discussion](https://github.com/relativitydev/relativity.testing.framework/discussions).
```
Comment (briantoms) [inline README.md]: Should this like to the RTF.API discussions instead?

File: README.md [hunk 2]
```diff
@@ -48,6 +79,14 @@ dotnet test ./Source/ --filter TestCategory=FunctionalTests -s .\DevelopmentScri
...
 ```
 
-## Documentation
+## Contributing
+
+See [CONTRIBUTING.md](https://github.com/relativitydev/relativity.testing.framework/blob/master/CONTRIBUTING.md).
+
+## Reporting Issues
+
+For feature enhancements or bug reports, please create an [Issue](https://github.com/relativitydev/relativity.testing.framework/issues) using the respective template. Before reporting an issue, please follow these guidelines. It helps us in understanding the request and provide a quicker response time.
```
Comment (briantoms) [inline README.md]: Should the issue links be to RTF.API issues instead?

**Review by briantoms — APPROVED** [review]

**Comment by Sai-Nissankula** [comment]
@paulletang , this was the parity PR I was talking about. I think it would be good to establish a pattern so that external developers dont find an incongruent experience.

**Comment by paulletang** [comment]
> @paulletang , this was the parity PR I was talking about. I think it would be good to establish a pattern so that external developers have a congruent experience. I think ideally this should of been 1 of the tasks or part of the tasks of updating the readme but I think that just mentioned only RTF repo and not the api one.

I've now seen what looks like the same changes (or followup changes) go into what feels like 6 different PRs.

We need to give more consideration to the scope of the work during planning/workshop to provide better acceptance criteria and tasks.

**Pull request merged**